frontend/src/components/CreateModal.jsx

```diff
@@ -0,0 +1,133 @@
+    this.setState({
+      offices: [...this.state.offices, {
+        key: (this.state.offices.length + 1).toString(),
+        name: office_name,
+        street,
+        city,
+        postal_code,
+        monthly_rent,
+        headquarter
+      }]
+    },()=>{
+      console.log(this.state.offices)
+
+      this.setState({
+        office_name: '',
+        street: '',
+        city: '',
+        postal_code: '',
+        monthly_rent: '',
+        headquarter: false
+      })
+    })
+  }
+
+  columns = [
+    {
+      title: 'Office Name',
+      dataIndex: 'name',
+      key: 'name',
+      render: text => <span>{text}</span>
+    },
+    {
+      title: 'Street',
+      dataIndex: 'street',
+      key: 'street',
+      render: text => <span>{text}</span>
+    },
+    {
+      title: 'Other',
+      dataIndex: 'tag',
+      key: 'headquarter',
+      render: (value, record) => {
+        if (record.headquarter) {
+          return (
+            <Tag color="green" >
+              Headquarter
+            </Tag>
+          )
+        }
+
+      },
+    }
+  ]
+
+  handleCancel = () => {
+    this.setState({
+      offices: []
+    }, () => {
+      this.props.onCancel()
+    })
+  }
+
+  handleOk = async () => {
+    const { company_name, offices } = this.state
+    const { onSave } = this.props
+    await axios.post(api_endpoints.CREATE_COMPANY_OFFICE, {
+      name: company_name,
+      offices
+    })
+    onSave()
+  }
+
+  render() {
+    const { ...props } = this.props
+    const { office_name, city, street, postal_code, monthly_rent, headquarter, offices } = this.state
+    return (
+      <Modal
+        {...props}
+        onCancel={() => this.handleCancel()}
+        onOk={() => this.handleOk()}
+      >
+        <p>
+          <b>Company Name:</b>  <br/>
+          <Input placeholder='Company Name' onChange={({ target: { value: company_name } }) => this.setState({ company_name })} />
+        </p>
+        <b>Headquarter:</b>
+        <div style={{display: "flex", justifyContent: 'space-around', marginTop: 5}}>
+          <Input style={{ width: "45%" }} placeholder='Office Name' value={ office_name } onChange={({target: { value: office_name }}) => this.setState({ office_name })}/>
+          <Input style={{ width: "45%" }} placeholder='Street' value={ street } onChange={({ target: { value: street } }) => this.setState({ street })}/>
+        </div>
+        <div style={{display: "flex", justifyContent: 'space-around', marginTop: 5}}>
+          <Input style={{ width: "45%" }} placeholder='City' value={ city } onChange={({ target: { value: city } }) => this.setState({ city })}/>
+          <Input style={{ width: "45%" }} placeholder='Postal Code' value={ postal_code }  onChange={({ target: { value: postal_code } }) => this.setState({ postal_code })}/>
+        </div>
+        <div style={{display: "flex", justifyContent: 'space-around', marginTop: 5}}>
```

Comment:
Seems like you don't use eslint?

Reply:
You are correct. I didn't use Eslint, here. Will figure it out in the next PR.